src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -3633,7 +3633,7 @@ public final class TokenTypes {
     public static final int LOR = JavaLanguageLexer.LOR;
     /**
      * The {@code &&} (conditional AND) operator.
-     *
+     * <p>
```

Comment:
I recommend to move this change in separate PR to merge sooner and keep this PR small and focused on main issue

Reply:
There are a few more violations, so we can do this in a separate PR